.github/workflows/e2e-arm64.yaml

```diff
@@ -0,0 +1,40 @@
+name: E2E-arm64
+on: [push, pull_request]
```

Comment:
Are you going to change this to a cron job after all workflow checks are green?

Reply:
Yes absolutely - I just want evidence that they are working as expected upfront 🙏🏻